### PR TITLE
Remove tarampampam packages

### PR DESCRIPTION
**Pull request by malyg1n**
## Description

Removed dependency `tarampampam/wrappers-php` because this package was deprecated and removed

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I wrote unit tests for my code _(if tests is required for my changes)_
- [x] I have made changes in `CHANGELOG.md` file

### Removed

Dependency `tarampampam/wrappers-php` because this package was deprecated and removed

### CODE REVIEW

File: src/ServiceProvider.php [hunk 1]
```diff
@@ -63,7 +60,7 @@ protected function getCredentials(ConfigRepository $config): array
                 throw new InvalidArgumentException('file does not exist');
             }
 
-            $credentials = (array) Json::decode((string) \file_get_contents($credentials_path));
+            $credentials = (array) \json_decode((string) \file_get_contents($credentials_path), true);
```
Comment (eldario) [inline src/ServiceProvider.php:63]:
```
$credentials = (array) \json_decode((string) \file_get_contents($credentials_path), true, 512, JSON_THROW_ON_ERROR);
```

File: src/ServiceProvider.php [hunk 2]
```diff
@@ -47,7 +45,6 @@ public function register(): void
      *
      * @param ConfigRepository $config
      *
-     * @throws JsonEncodeDecodeException If config file with credentials has invalid JSON format
```
Comment (eldario) [inline src/ServiceProvider.php:50]: \JsonException

File: composer.json [hunk 3]
```diff
@@ -23,8 +23,7 @@
         "illuminate/notifications": "~6.0 || ~7.0 || ~8.0",
```
Comment (eldario) [inline composer.json:23]: php 7.3

**Review by eldario — CHANGES_REQUESTED** [review]

File: src/ServiceProvider.php (hunk 2 above)
Comment (malyg1n) [reply src/ServiceProvider.php:50]: done

File: composer.json (hunk 3 above)
Comment (malyg1n) [reply composer.json:23]: already

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## Unreleased
```
Comment (eldario) [inline CHANGELOG.md:7]:
```suggestion
## v2.5.0
```

**Review by eldario — APPROVED** [review]

**Pull request merged**